pom.xml

```diff
@@ -132,7 +132,7 @@
         <scopt.version>4.0.1</scopt.version>
         <slf4j.version>1.7.35</slf4j.version>
         <log4j.version>2.17.1</log4j.version>
-        <spark.version>3.1.2</spark.version>
+        <spark.version>3.2.1</spark.version>
```

Comment:
we should also change `delta.version`, because the minor version of delta and spark is one-one mapping

Reply:
we can left some comments here, e.g. 'if change minor version of spark, don't forget to change the deafult `delta.version`'